### ISSUE TITLE
cosmic-silverblue: GNOME Wayland not launching

### ISSUE THREAD

**Issue by boredsquirrel** [ublue-os/cosmic#25]
from cosmic greeter I can not launch GNOME on Wayland. Last time I checked only GNOME on Xorg worked.

I will try from tty with `gnome-session --session gnome-wayland`

**Comment by dylanmtaylor** [comment]
Are these images intended to have more than one desktop environment? I think we would only have Cosmic in the final version.

**Comment by etrigan63** [comment]
> Are these images intended to have more than one desktop environment? I think we would only have Cosmic in the final version.

I agree. The upstream project is Fedora Cosmic Atomic so this one should be Ublue Cosmic.

**Comment by dylanmtaylor** [comment]
From what I understand, those desktops are temporary and will go away from these images, long-term

**Comment by boredsquirrel** [comment]
This would be the only ublue image featuring 2 desktops. I suppose the drop will be when COSMIC is stable, not alpha and not beta anymore.
